BasiliskII/src/CrossPlatform/sigsegv.cpp

```diff
@@ -342,12 +342,12 @@ static void powerpc_decode_instruction(instruction_t *instruction, unsigned int
 #define SIGSEGV_REGISTER_FILE			(unsigned long *)&SIGSEGV_CONTEXT_REGS->nip, (unsigned long *)(SIGSEGV_CONTEXT_REGS->gpr)
 #define SIGSEGV_SKIP_INSTRUCTION		powerpc_skip_instruction
 #elif (defined(arm) || defined(__arm__))
-#define SIGSEGV_CONTEXT_REGS			(((struct ucontext *)scp)->uc_mcontext)
+#define SIGSEGV_CONTEXT_REGS			(((ucontext_t *)scp)->uc_mcontext)
```

Comment:
Don't we need the old code if we're compiling against an older glibc? Can it be made conditional on glibc version?